OWNERS_ALIASES

```diff
@@ -1,13 +1,22 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 aliases:
-  operator-reviewers:
-  - zimnx
-  - rzetelskik
-  - mflendrich
+  operator-team:
+  # All members of the team.
   - czeslavo
-  operator-approvers:
+  - mflendrich
+  - rzetelskik
   - zimnx
+  operator-review-autoassignees:
+  # Subset of operator-team that will be automatically assigned to PRs.
```

Comment:
I think this way it will be unambiguous - "being assigned" is not equal to "being requested for review" in the GH PRs world. 
```suggestion
  # Subset of operator-team that will be automatically requested for reviews in PRs.
```

Reply:
Done.